eng/smoke-tests/src/main/java/com/azure/App.java

```diff
@@ -0,0 +1,25 @@
+// ------------------------------------
```

Comment:
The MIT header for our java files is the one below. This should be added to all those files down there.

```java
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```